audit/projects/k8s-gcr-audit-test-prod/services/pubsub/subscriptions.json

```diff
@@ -1,18 +0,0 @@
-[
-  {
-    "ackDeadlineSeconds": 10,
-    "expirationPolicy": {},
-    "messageRetentionDuration": "604800s",
-    "name": "projects/k8s-gcr-audit-test-prod/subscriptions/cip-auditor-test-invoker",
```

Comment:
Not entirely sure why this is getting deleted...? Is this a subscription age thing or has the auditor stopped running?